monster_stats.js

```diff
@@ -3493,7 +3493,7 @@ MONSTER_STATS = {
             "attack": 2,
             "range": 2,
             "attributes": [
-              "%shield% 1"
+              "%flying%","%shield% 1"
```

Comment:
No particular reason I can remember. I probably did not put enough attention to implicit syntax convention. Apologies.